crates/bevy_asset/src/lib.rs

```diff
@@ -1,3 +1,106 @@
+//! In the context of game development, an "asset" is a piece of multimedia content which must be loaded from disk and displayed in the game.
+//! Typically, these are authored by artists and designers (in contrast to code),
+//! are relatively large in size, and include everything from textures and models to sounds and music to levels and scripts.
```

Comment:
Maybe don't say scripts, given Bevy's focus on Rust and ECS. Up to you.

Reply:
I think its reasonable to say scripts here in the context of defining the range of what an asset might be. In that context I don't think we need to constrain ourselves to what Bevy currently supports upstream.

Reply:
Yep, my primary motivation for including it was to expand people's mindset on "what can an asset be": it's not just sounds and graphics!